config/initializers/secure_headers.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 if Rails.env.development?
   SecureHeaders::Configuration.default do |config|
```

Comment:
This file is *terrible* as default. I started a project and had a *ton* of modifying to do so I could get *anything* to load at all. We should develop this file more fully and then comment the crap out of it. This is what I wound up with so I could use things *somewhat* normally:

```ruby
SecureHeaders::Configuration.default do |config|
  config.cookies = {
    secure: true,
    httponly: true,
    samesite: {
      lax: true
    }
  }
  config.hsts = "max-age=#{1.week.to_i}"
  config.x_frame_options = 'SAMEORIGIN'
  config.x_content_type_options = 'nosniff'
  config.x_xss_protection = '1; mode=block'
  config.x_download_options = 'noopen'
  config.x_permitted_cross_domain_policies = 'none'
  config.referrer_policy = %w(origin-when-cross-origin strict-origin-when-cross-origin)
  config.csp = config.csp.merge(
    default_src: %w('none'),
    connect_src: %w('self'),
    script_src: %w('self' 'unsafe-inline' 'unsafe-eval' http: https:),
    font_src: %w(http: https:),
    style_src: %w('self' 'unsafe-inline' http: https:),
  )
end
```

Reply:
I added these wholesale, we can reevaluate later.